tsconfig.integration.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "./tsconfig.spec.json",
+  "include": ["public/cli/lib/rendering/**/*.ts"],
```

Comment:
I'm ready to make the include broader, to the whole public/cli folder, which I'll do on another branch.